src/request/caches/modes.rs

```diff
@@ -0,0 +1,32 @@
+use serde::{Deserialize, Serialize};
+
+const DEFAULT_CONCURRENCY_LEVEL: i32 = 1000;
+const DEFAULT_ACQUIRE_TIMEOUT: i32 = 15000;
+
+#[derive(Serialize, Deserialize)]
+pub struct LocalCache {
+    locking: Locking,
+    statistics: bool,
+}
+
+#[derive(Serialize, Deserialize)]
+struct Locking {
+    #[serde(rename = "concurrency-level")]
+    concurrency_level: i32,
+    #[serde(rename = "acquire-timeout")]
+    acquire_timeout: i32,
+    striping: bool,
+}
```

Comment:
I know this is _just_ moving this struct, but FYI:

`#[serde(rename_all = "kebab-case")]`

Reply:
Fixed :+1: